packages/hadron-build/commands/verify.js

```diff
@@ -1,53 +0,0 @@
-'use strict';
-/**
- * Wouldn't it be great if you or a CI system were notified properly
- * that you aren't using the right version of node.js or npm?
- *
- * @see https://github.com/atom/atom/blob/master/script/utils/verify-requirements.js
- */
-
-const semver = require('semver');
-const cli = require('mongodb-js-cli')('hadron-build:verify');
-const { promisify } = require('util');
-const run = require('./../lib/run');
-const runAsync = promisify(run);
-
-exports.command = 'verify [options]';
-exports.describe = 'Verify the current environment meets the app\'s requirements.';
-
-exports.builder = {
-  nodejs_version: {
-    describe: 'What version of node.js is required for this app?',
-    default: '^7.4.0'
```

Comment:
Drive-by, just caught my eye how out of date this was, so I removed this script too